### ISSUE TITLE
Unable to like, repost or reply to feed posts using a screen reader (mobile app)

### ISSUE THREAD

**Issue by hs4man21** [bluesky-social/social-app#685]
**Describe the bug**
Using [the VoiceOver screen reader on my iPhone](https://build.washingtonpost.com/resources/accessibility/screen-readers#iOS), I am unable to like, repost or reply to posts in my feed because I can only access the top-level link and none of the nested touchable/buttons within it. I should be able to "swipe down" on any given post to access the post's nested buttons, whereas swiping "left to right" steps me between posts at the top level.

Twitter does have the desired functionality (demonstrated in the second video below) and could be used as a model for how this would work.

This also applies to the nifty "Follow" button embedded in feed posts (and on user profile cards in the search tab).

**To Reproduce**

1. Use an iPhone (although I believe Android devices have this same issue when using their screen reader, TalkBack).
2. Turn on [the VoiceOver screen reader](https://build.washingtonpost.com/resources/accessibility/screen-readers#iOS) in your iPhone's accessibility settings.
3. Open the Bluesky app.
4. Tap a feed post in the app to hear it read by VoiceOver, as shown below (turn on your audio as the videos below have volume).
5. Note that if you swipe down to attempt to reach the buttons within the feed post, you are unable. 

**Expected behavior**

Using Twitter as an example, I expect to have the option to swipe down to access nested buttons such as the like, repost and comment buttons. Twitter is not perfect, but parity with it would be a great step in the right direction for accessibility.

**Screenshots**

(Turn on audio to hear VoiceOver reading the items. You can also hear the sound from me trying to swipe into a post's buttons.)

https://github.com/bluesky-social/social-app/assets/38192823/e92289ee-318e-4293-a791-56b43df9ea61

Twitter example, for comparison (note ability to swipe into the post and "like" it here, in addition to hearing the likes at top level):

https://github.com/bluesky-social/social-app/assets/38192823/b154950a-3b3d-404d-a12a-c7d88ed9030b

**Details**

- Platform: iPhone XR, iOS 16.4.1, using VoiceOver screen reader
- Platform version: MacOS: Ventura 13.1, Chrome: 112.0.5615.137
- App version: 1.28

**Comment by renahlee** [comment]
👋  previously confirmed with Paul this is a task I'm working on this week ❤️

**Comment by ansh** [comment]
This was fixed in #765
